Dockerfile

```diff
@@ -1,6 +1,6 @@
 # =================== BASE BUILD LAYER ===================
 # this layer is used to prepare a common layer for both debug and release builds
-FROM golang:1.13 as secrets-provider-builder-base
+FROM golang:1.15 as secrets-provider-builder-base
```

Comment:
does this change need to be in the CHANGELOG? does it affect the user in any way?

Reply:
it is already in the changelog, but I don't think it has a user impact in any case